tests/unit/conftest.py

```diff
@@ -202,3 +203,11 @@ def mocked_create_identity(mocker: MockerFixture, kratos_identity_json: Dict) ->
 def mocked_run_migration(mocker: MockerFixture) -> MagicMock:
     mock = mocker.patch("charm.KratosAPI.run_migration", return_value=(None, None))
     return mock
+
+
+@pytest.fixture()
```

Comment:
Perhaps it would make sense to set the `autouse=True` flag, so that this is mocked in all tests to avoid fetching the promtail binary by accident